xml/docu_styleguide.structure.xml

```diff
@@ -421,11 +421,15 @@ default 0 <co xml:id="co.default"/></screen>
 <screen>To start LibreOffice from the command line, use
 &lt;command&gt;loffice&lt;/command&gt;.</screen>
    <para>
-    Where options belong to a command, add them after the command markup and a
-    space character using the element <tag class="emptytag">option</tag>:
+    Where options or subcommands belong with a command, include
+    them within the element <tag class="emptytag">command</tag> itself:
    </para>
 <screen>To start LibreOffice Writer from the command line, use
-&lt;command&gt;loffice&lt;/command&gt; &lt;option&gt;--writer&lt;/option&gt;.</screen>
+&lt;command&gt;loffice --writer&lt;/command&gt;.</screen>
```

Comment:
why not place `--writer` *outside* of `</command>`?

I have a counter example. In our [crmsh help section](https://susedoc.github.io/doc-sleha/develop/SLE-HA-guide/html/cha.ha.manual_config.html#sec.ha.manual_config.crm.help), we need to distinguish between just the command name and other things.

It doesn't make sense to put everything in `command` as it would dilute the message.

Reply:
As I/Frank said multiple times already, the goal is to separate input and output on the command line.

Reply:
And you can still have exceptions, if they are really required.